### PR TITLE
Feature/add assess stud reg

### PR DESCRIPTION
**Pull request by eckermania**

### CODE REVIEW

File: backend/src/components/eas/eas.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ async function getAssessmentSessionsBySchoolYear(req, res) {
     const url = `${config.get('server:eas:assessmentSessionsURL')}/school-year/${req.params.schoolYear}`;
     let data = await getData(url);
     data.forEach(session => {
-      session.isOpen =  new Date(session.activeFromDate) <= new Date() && new Date(session.activeUntilDate) >= new Date(); 
+      session.isOpen =  new Date(session.activeFromDate) <= new Date() && new Date(session.activeUntilDate) >= new Date();
```
Comment (arcshiftsolutions) [inline backend/src/components/eas/eas.js:29]: Please use LocalDate

File: frontend/src/utils/institute/formRules.js [hunk 2]
```diff
@@ -189,6 +189,10 @@ const minLength = ( length, message = 'Must be at least ${length} characters') =
   return v => !v || v.length === length || message;
 };
 
+const penIsValid = (message = 'PEN must be a 9 digit number') => {
```
Comment (arcshiftsolutions) [inline frontend/src/utils/institute/formRules.js:192]: Are we including check digit validation?

Comment (eckermania) [reply frontend/src/utils/institute/formRules.js:192]: if you are talking about checking for a 9 digit number, that's what the regex in the next line is doing. Are you referring to the same validCheckDigit in the PenUtil in the api? If so, I would need to add this.

Comment (arcshiftsolutions) [reply frontend/src/utils/institute/formRules.js:192]: Yes that's right the `validCheckDigit`. Ideally we layer it on to ensure they're typing in an actual PEN and not just any 9 digit number.

**Review by arcshiftsolutions — APPROVED** [review]

**Pull request merged**